core/ir/x86/opcode_api.h

```diff
@@ -1692,4 +1692,8 @@ enum {
 #define OP_icebp OP_int1
 #define OP_setalc OP_salc
 
+/* Renamed opcodes. */
+#define OP_ud2a OP_ud2
```

Comment:
Indeed, you have no Doxygen comment here -- this is why Doxygen couldn't add the link.